api/cases/case.js

```diff
@@ -5,17 +5,23 @@ const valueProcessor = require('../lib/processors/value-processor');
 const { getEvents } = require('../events');
 const { getDocuments } = require('../documents');
 const { getAllQuestionsByCase } = require('../questions');
+const getCaseTemplate = require('./templates/case');
 
 function getCase(caseId, userId, options, caseType = 'Benefit', jurisdiction = 'SSCS') {
     return generateRequest('GET', `${config.services.ccd_data_api}/caseworkers/${userId}/jurisdictions/${jurisdiction}/case-types/${caseType}/cases/${caseId}`, options);
 }
 
 function getCaseWithEventsAndQuestions(caseId, userId, options, caseType, jurisdiction) {
-    return Promise.all([
+    const promiseArray = [
         getCase(caseId, userId, options, caseType, jurisdiction),
-        getEvents(caseId, userId, options, caseType, jurisdiction),
-        getAllQuestionsByCase(caseId, userId, options, jurisdiction)
-    ]);
+        getEvents(caseId, userId, options, caseType, jurisdiction)
+    ];
+
+    if(jurisdiction === 'SSCS') {
```

Comment:
we have a function hasCOR and then we have this? please could we use the function, rather than having multiple instances of how we decide to include cor.

Reply:
this will allow us to refactor more efficiently when we get the data from idam.

Reply:
👍 will addhasCOR function

Reply:
maybe to future proof would it be an idea to return a promise.resolve([]) if it doesn't have COH
then it mean if we need to do another call to another microsevers at the same time we would just be extending the returning array `        .then(([caseData, events, questions]) => {`